src/github.ts

```diff
@@ -194,12 +201,12 @@ export const rebase = async (
 		await ctx.octokit.git.createRef({
 			...ctx.repo(),
 			ref: `refs/heads/${tempBranchName}`,
-			sha: baseCommit.sha,
+			sha: baseSha,
 		});
 
 		// Cherry-pick each commit
-		let currentSha = pr.base.sha;
```

Comment:
So the issue was here right? Using `pr.base.sha` instead of the `.sha` property returned from [getCommit](https://docs.github.com/en/rest/commits/commits?apiVersion=2022-11-28#get-a-commit)